buildsystem/cmake-includes/silence-warnings/03-all/clang/11-uninitialized-const-pointer/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# In an optimal state, this file does not exist
+# XXX - The leading spaces in this file are meaningful for string concatenation!
```

Comment:
In general, I would prefer to use something like "WARNING" here to avoid vaguely racy strings and improve clarity. However, it not enough to warrant rejecting this PR.